### PR TITLE
Feature init

### PR DESCRIPTION
**Pull request by olivierverwoerd**

### CODE REVIEW

File: src/moving_platform.hpp [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/**
```
Comment (ghost) [inline src/moving_platform.hpp:1]: All the @param[in] commands in this file need a parameter name.

File: src/moving_platform.hpp [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+#define MOVING_PLATFORM_HPP
+
+namespace MovingPlatform {
+/**
+ * @brief Platform
+ * This class communicates with the motorcontroller to control multiple moters
+ * With this class a platforms movement can be set with ease
+ */
+class Platform {
+    /// This is designed to break. waiting for motorcontroller to
```
Comment (ghost) [inline src/moving_platform.hpp]: Useless comment.

File: src/moving_platform.hpp [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+ * @author    Olivier verwoerd & Maarten Wassenaar
+ * @license   See LICENSE
+ */
+
+#ifndef MOVING_PLATFORM_HPP
+#define MOVING_PLATFORM_HPP
+
+namespace MovingPlatform {
+/**
+ * @brief Platform
```
Comment (ghost) [inline src/moving_platform.hpp]: Doxygen knows which class/function this comment paragraph corresponds to and shows that name as the title, so simply copying the name is useless.
Change this and all other @brief's in this file that have just the name to a more descriptive sentence.

**Review by ghost — CHANGES_REQUESTED** [review]
Looks good besides some small doxygen errors.

**Comment by DanielvanVliet** [comment]
I fixed the doxygen and updated the methods to fit the interface mentioned in the vertical slice. @JasperSmienk

**Comment by ghost** [comment]
@Danielvv1998 The @param[in] commands still need the name of the parameter.
e.g. `@param[in] <parameter> <description>`

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Everything is updated and fixed now, merging pull-request.

**Pull request merged**